modules/distribution-zones/src/integrationTest/java/org/apache/ignite/internal/distributionzones/ItIgniteDistributionZoneManagerNodeRestartTest.java

```diff
@@ -642,6 +653,14 @@ public void testCreationZoneWhenDataNodesAreDeletedIsNotSuccessful() throws Exce
         assertThat(metastore.get(new ByteArray(dataNodeKey[0])).thenApply(Entry::tombstone), willBe(true));
     }
 
+    private static Map<Node, Integer> deserializeDataNodesMap(byte[] bytes) {
```

Comment:
Why can't we use `DistributionZonesUtil#deserializeDataNodesMap`? The same with `deserializeLogicalTopologySet`, I see that you have several copies of this method in different test classes, could we us general method? We could add it to this class as well.